src/Classes/Utils/CanvasUtils.ts

```diff
@@ -23,49 +22,57 @@ export class CanvasUtils {
     }
 
     public static drawPolygonMask(context: CanvasRenderingContext2D,
-                                  rawData: ICoordinates[],
-                                  stroke: IPolygonMaskDrawStroke): void {
-        context.save();
-        context.beginPath();
-        context.moveTo(rawData[0].x, rawData[0].y);
+        rawData: ICoordinates[],
+        stroke: IPolygonMaskDrawStroke): void {
+        const color = typeof stroke.color === "string" ? ColorUtils.stringToRgb(stroke.color) : ColorUtils.colorToRgb(stroke.color);
 
-        for (let i = 1; i < rawData.length; i++) {
-            context.lineTo(rawData[i].x, rawData[i].y);
-        }
+        if (color) {
+            context.save();
+            context.beginPath();
+            context.moveTo(rawData[0].x, rawData[0].y);
 
-        context.closePath();
-        context.strokeStyle = stroke.color;
-        context.lineWidth = stroke.width;
-        context.stroke();
-        context.restore();
+            for (let i = 1; i < rawData.length; i++) {
+                context.lineTo(rawData[i].x, rawData[i].y);
+            }
+
+            context.closePath();
+            context.strokeStyle = ColorUtils.getStyleFromColor(color);
+            context.lineWidth = stroke.width;
+            context.stroke();
+            context.restore();
+        }
     }
 
     public static drawLineLinked(context: CanvasRenderingContext2D,
-                                 width: number,
-                                 begin: ICoordinates,
-                                 end: ICoordinates,
-                                 backgroundMask: boolean,
-                                 colorLine: IRgb,
-                                 opacity: number,
-                                 shadow: ILineLinkedShadow): void {
+        width: number,
+        begin: ICoordinates,
+        end: ICoordinates,
+        backgroundMask: boolean,
+        colorLine: IRgb,
+        opacity: number,
+        shadow: ILineLinkedShadow): void {
         context.save();
 
         if (backgroundMask) {
             context.globalCompositeOperation = 'destination-out';
         }
 
         if (colorLine) {
-            context.strokeStyle = `rgba(${colorLine.r},${colorLine.g},${colorLine.b},${opacity})`;
+            context.strokeStyle = ColorUtils.getStyleFromColor(colorLine, opacity);;
         }
 
         context.lineWidth = width;
         // this.ctx.lineCap = "round"; /* performance issue */
         /* path */
         context.beginPath();
-        if (shadow.enable) {
+
+        const color = typeof shadow.color === "string" ? ColorUtils.stringToRgb(shadow.color) : ColorUtils.colorToRgb(shadow.color);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)